src/analytics/README.md

```diff
@@ -1,32 +1,58 @@
 fetch branch from github.
-  - git fetch origin <branch-name>
-  - git checkout <branch-name>
+- `git fetch origin <branch-name>`
+- `git checkout <branch-name>`
 
 create local python environment
-  windows
-	python -m venv [local_env_name e.g env]
-	
-   linux/mac
-	python3 -m venv [local_env_name e.g env]
+
+windows
+
+`python -m venv [local_env_name e.g env]`
+
+linux/mac
+
+`python3 -m venv [local_env_name e.g env]`
 
 activate the environment
-	windows
-	- [local_env_name]\Scripts\activate
-	
-	linux/mac
-	- source [local_env_name]/bin/activate
+windows
+- `[local_env_name]\Scripts\activate`
+
+linux/mac
+- source `[local_env_name]/bin/activate`
 
 Install dependencies using the requirements.txt
-	-pip install -r requirements.txt
+- `pip install -r requirements.txt`
 
 set environment variables
- 	windows
- 		- set FLASK_APP=app.py
- 		- set FLASK_ENV=development
-    linux/mac
-		- export FLASK_APP=app.py
-		- export FLASK_ENV=development
-	
+windows
+- `set FLASK_APP=app.py`
+- `set FLASK_ENV=development`
+linux/mac
+- `export FLASK_APP=app.py`
+- `export FLASK_ENV=development`
+
 
 Run the Flask App
-    - flask run
\ No newline at end of file
+- `flask run`
+
+# Data sources
```

Comment:
Thanks @bobturneruk  for this.

- Users/Collabs/Defaults/Candidates are currently not being used because we transitioned to other databases for different organisations (AirQo and KCCA). These are: **airqo_netmanager_airqo** and **airqo_netmanager_kcca**. 

Naming convention is: {DATABASE_NAME}_{ORGANISATION_NAME}

Reply:
Also note that there is another [open PR](https://github.com/airqo-platform/AirQo-api/pull/177) where most of the work in this microservice is being refactored.

Reply:
Thanks @Baalmart!